### PR TITLE
Fixes RMSF docstring (Issue #2806)

### LINKED ISSUE #2806: Typo in RMSF docstring
## Expected behavior ##

Following examples in docstring should work.

## Actual behavior ##

I believe there is a typo in https://github.com/MDAnalysis/mdanalysis/blob/343e10a257690e4312673f2a0fa4703b31b2e223/package/MDAnalysis/analysis/rms.py#L741

The reference is missing, it probably should be `align.AlinTraj(u, u, select...)` (as per the PMDA docstring)?

## Current version of MDAnalysis ##

develop doctstring

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #2806 

Changes made in this Pull Request:
 - Fixes the RMSF docstring's align command and adds transformation to make the results accurate.

CHANGELOG entry here is probably unnecessary.

PR Checklist
------------
 - Tests?
 - [x] Docs?
 - CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/analysis/rms.py [hunk 1]
```diff
@@ -745,27 +745,37 @@ def __init__(self, atomgroup, **kwargs):
            u = mda.Universe(TPR, XTC, in_memory=True)
            protein = u.select_atoms("protein")
 
-           # 1) need a step to center and make whole: this trajectory
-           #    contains the protein being split across periodic boundaries
-           #
-           # TODO
+           # 1) the current trajectory contains a protein split accross
```
Comment (fiona-naughton) [inline package/MDAnalysis/analysis/rms.py:748]: accross -> across

**Review by fiona-naughton — CHANGES_REQUESTED** [review]
One little spelling error but otherwise lgtm :)

**Comment by IAlibay** [comment]
Ok good, looks like CI works 🎉 

P.S. I need to fix @fiona-naughton's comment, but this will need to wait until #3046 since it involves docs.

**Review by fiona-naughton — APPROVED** [review]

**Comment by lilyminium** [comment]
Thank you @IAlibay!

**Pull request merged**